scripts/pi-hole/php/header.php

```diff
@@ -338,7 +338,7 @@ function pidofFTL()
                         }
 
                         // CPU Temp
-                        if($FTL)
```

Comment:
This was set in #408 with the intention of hiding the temp display if FTL was offline. 

>Hide temperature if FTL is not running and show FTL status